src/XdebugHandler.php

```diff
@@ -64,8 +66,14 @@ public function __construct($envPrefix)
         if (extension_loaded('xdebug')) {
             $ext = new \ReflectionExtension('xdebug');
             $this->loaded = $ext->getVersion() ?: 'unknown';
+
+            if (false !== ($mode = ini_get('xdebug.mode'))) {
+                $this->mode = getenv('XDEBUG_MODE') ?: ($mode  ?: 'off');
+            }
         }
 
+        self::$xdebugOff = $this->mode === 'off';
```

Comment:
If I get this right, if xdebug is not loaded, xdebufOff will be false, which is rather misleading.

As an aside, I'd make the getter isXdebugOff => isXdebugEnabled or isXdebugActive or something, to avoid double negations.

Reply:
Yes, I'm getting into a tangle here. The intent was to provide a method to determine if Xdebug is present because it is running with xdebug.mode=off (`extension_loaded('xdebug') && XdebugHandler::isXdebugOff()`)  - in the absence of an official method to get the mode.

However, I like isXdebugActive(). This could return:

- **true** if Xdebug is loaded and, if it supports modes, is running in a mode that enables it to do something
- **false** if Xdebug is not loaded, or it is running with xdebug.mode=off 

Does this sound like a plan?

Reply:
Yup that sounds much clearer to me, and allows people to use this without caring about extension_loaded checks too 👍🏻

Reply:
Okay, I've changed isXdebugOff() to isXdebugActive()